src/main.rs

```diff
@@ -810,13 +812,16 @@ fn verify_files_main(
         false,
     )?;
 
+    // Find 'looping' procs i.e. procs that contain a proc call which results in a recursive loop.
```

Comment:
I think it's valuable to stress that the analysis finds *potentially looping* calls (it over-approximates).